src/lib/mina_ledger/ledger.ml

```diff
@@ -737,3 +868,43 @@ let%test_unit "zkapp_command application on masked ledger" =
           assert (not (Ledger_hash.equal init_merkle_root (L.merkle_root l))) ;
           (*Parent updates reflected in child masks*)
           assert (Ledger_hash.equal (L.merkle_root l) (L.merkle_root m)) ) )
+
+let%test_unit "user_command application on converting ledger" =
+  let open Mina_transaction_logic.For_tests in
+  let module L = Ledger_inner in
+  let constraint_constants =
+    { Genesis_constants.For_unit_tests.Constraint_constants.t with
+      account_creation_fee = Currency.Fee.of_nanomina_int_exn 1
+    }
+  in
+  Quickcheck.test ~trials:1 Test_spec.gen ~f:(fun { init_ledger; specs } ->
```

Comment:
I just copied the specs from the `user_command` tests because I saw that they create/modify accounts. Since I don't care about what the account data actually is, only that it is there and duplicated to the converting ledger, i figured this was ok.